distribution/rules.bzl

```diff
@@ -16,38 +16,30 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
 
+load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar", "pkg_deb")
+load("@bazel_tools//tools/build_defs/pkg:rpm.bzl", "pkg_rpm")
+
 def _java_deps_impl(ctx):
     names = {}
     files = []
     newfiles = []
 
-    for target in ctx.attr.targets:
-        for file in target.data_runfiles.files.to_list():
-            if file.extension == 'jar':
-                names[file.path] = ctx.attr.java_deps_root + file.basename
-                files.append(file)
-
-
-    java_deps_builder = ctx.actions.declare_file('_java_deps.py')
-
-    ctx.actions.expand_template(
-        template = ctx.file._java_deps_builder,
-        output = java_deps_builder,
-        substitutions = {
-            "{moves}": str(names),
-            "{distribution_tgz_location}": ctx.outputs.distribution.path,
-        },
-        is_executable = True
-    )
+    for file in ctx.attr.target.data_runfiles.files.to_list():
+        if file.extension == 'jar':
+            names[file.path] = ctx.attr.java_deps_root + file.basename
+            files.append(file)
 
     ctx.actions.run(
         outputs = [ctx.outputs.distribution],
         inputs = files,
-        executable = java_deps_builder
+        arguments = [str(names), ctx.outputs.distribution.path],
+        executable = ctx.file._java_deps_builder,
+        progress_message = "Generating tarball with Java deps: {}".format(
+            ctx.outputs.distribution.short_path)
     )
 
 
-def _distribution_impl(ctx):
+def _old_distribution_impl(ctx):
```

Comment:
is this still used? if it's just a dead-code, let's remove it.

Reply:
@lolski ok